src/SpeedDial/speeddial.st.css

```diff
@@ -0,0 +1,63 @@
+@st-import [--spacing-unit] from "../styles/default/project.st.css";
+@namespace "SpeedDial";
+
+.root {
+  -st-states:
+    speedDialOpen(string);
+  display: flex;
+}
+
+.root > .buttonGroup {
+  margin-left: var(--spacing-unit);
+}
+
+.root:speedDialOpen(false) > div > .addButton > svg {
```

Comment:
It is better practice to only use direct selectors with `::ClassName` rather than targeting HTML elements if possible.